exotations/task_maps/task_map/src/JointVelBackFiniteDiff.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ *      Author: Christopher E. Mower
+ *
+ * Copyright (c) 2018, University Of Edinburgh
```

Comment:
Please apply comments from other files to this one - it is the same places/things that need to be updated